spec/models/card_spec.rb

```diff
@@ -146,5 +146,21 @@
       end
     end
   end
+
+  context 'check levenshtein distance' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.